build/expect.customize.js

```diff
@@ -0,0 +1,41 @@
+import expect from 'expect';
+import jest from "jest-mock";
```

Comment:
Hi, i would call this JestMock, what do you think?

Reply:
Is this a temporary solution that we are going to remove if/when we migrate to jest ??

Reply:
I tried to mantain compability where it made sense: spyOn is now part of jest, not of expect anymore, but for now I would not make jest presence explicit.
About the name, yes, np.